packages/rlc-common/src/metadata/buildPackageFile.ts

```diff
@@ -288,6 +291,17 @@ function restLevelPackage(model: RLCModel, hasSamplesGenerated: boolean) {
       packageInfo.scripts["integration-test:node"] =
         "dev-tool run test:node-js-input -- --timeout 5000000 'dist-esm/test/**/*.spec.js'";
     }
+    if (isTypeSpecTest) {
```

Comment:
I think the condition should be `!azureSdkForJs` because we will get rid of dev-tool dependency not in sdk repo, but not only in typespec test env.

Reply:
This is using the same condition above where we set `"type": "module"` for which we need to add the mocha `ts-node/esm` loader option.  There's no dev-tool or js repo specific code here. The difference is that `-r esm --require ts-node/register` is not needed.